client/components/image/index.tsx

```diff
@@ -0,0 +1,26 @@
+import classnames from 'classnames';
+import { SyntheticEvent } from 'react';
+
+const hideImageOnError = ( event: SyntheticEvent< HTMLImageElement > ) => {
+	event.currentTarget.style.display = 'none';
+};
+
+export default function Image( { alt, className, src, ...rest }: ImageProps ) {
```

Comment:
I think we should still switch to this type which gives more correct typings since this component is designed to take all the same props as a `<img>` tag.

```suggestion
export default function Image( { alt, className, src, ...rest }: React.ImgHTMLAttributes< HTMLImageElement >  ) {
```

Reply:
Oops, I missed @zaguiini's [comment above](https://github.com/Automattic/wp-calypso/pull/66061#discussion_r939842724). His suggestion is better!